### PR TITLE
add a cli command to generate a DBML dump

### PR DESCRIPTION
**Pull request by ewdurbin**
This has some pretty immediate "byproduct" wins:

- We can map it: https://dbdiagram.io/d/645a51c8dca9fb07c4c56c19
- We can generate docs: https://dbdocs.io/ewdurbin/PyPI

My usecase is to automatically generate and compare a [fides dataset](https://ethyca.github.io/fideslang/resources/dataset/) using the output though...

### CODE REVIEW

**Comment by ewdurbin** [comment]
Example of how the result could be used to generate/merge-into an existing fides dataset: https://gist.github.com/ewdurbin/05914fee6eb4e53d4357fb3b27f76b36#file-0-dbml_to_fides_dataset-py

File: requirements/main.in [hunk 1]
```diff
@@ -55,6 +55,7 @@ readme-renderer[md]>=36.0
 requests
 requests-aws4auth
 redis>=2.8.0,<5.0.0
+rich
```
Comment (miketheman) [inline requirements/main.in:58]: I _think_ we could replace most, if not all, of the `rich` usage with `click` builtins that we already use. See:

https://click.palletsprojects.com/en/8.1.x/utils/#printing-to-stdout
https://click.palletsprojects.com/en/8.1.x/api/#click.echo

https://click.palletsprojects.com/en/8.1.x/utils/#ansi-colors
https://click.palletsprojects.com/en/8.1.x/utils/#intelligent-file-opening

**Review by miketheman — COMMENTED** [review]
I generally like it, but would like to see if leveraging click can get us there without adding more dependencies to the production stack. I recognize these aren't runtime concerns, but let's see how far we can push Click.

File: requirements/main.in (hunk 1 above)
Comment (ewdurbin) [reply requirements/main.in]: [b075088](https://github.com/pypi/warehouse/pull/13615/commits/b0750885fde3dc9edadf2c826830c572e81c6e27)

**Review by miketheman — APPROVED** [review]
Thanks for working through this!

**Pull request merged**